repo2docker/buildpacks/base.py

```diff
@@ -358,6 +387,8 @@ def render(self):
             build_script_files=self.build_script_files,
             base_packages=sorted(self.base_packages),
             post_build_scripts=self.post_build_scripts,
+            entrypoint_json=json.dumps(self.entrypoint),
+            command_json=json.dumps(self.command)
```

Comment:
minor style nit: including trailing commas on lists/dicts gives a better diff when adding/removing items on the end of things.